client/lib/explat/internals/log-error.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { getLogger } from 'calypso/server/lib/logger';
+
+// Using typescript to ensure we are being safe in SSR
+declare const window: undefined | ( Window & typeof globalThis );
+
+export const isDevelopmentMode = process.env.NODE_ENV === 'development';
+
+/**
+ * Log an error to the server.
+ * Works in SSR.
+ *
+ * Should never throw.
+ *
+ * @param error Error to save
+ */
+export const logError = (
+	error: Record< string, string > & { message: string; properties?: Record< string, unknown > }
+): void => {
+	try {
+		error[ 'properties' ] = {
+			...( error?.[ 'properties' ] || {} ),
+			context: 'explat',
+			explat_client: 'calypso',
+		};
+
+		if ( typeof window === 'undefined' ) {
+			// Bunyan logger will log to the console in development mode.
+			getLogger().error( error );
+		} else {
+			if ( isDevelopmentMode ) {
```

Comment:
Are we intentionally sending the error to the server in development mode? If so, is there a way to differentiate development errors from production errors in the server logs?

Reply:
Ah I can disable sending them to the server in production.